functions/auth.js

```diff
@@ -3,7 +3,7 @@ const admin = require("firebase-admin");
 const serviceAccount = require("./private/cl-dev-pk.json");
 admin.initializeApp({
   credential: admin.credential.cert(serviceAccount),
-  databaseURL: process.env.REACT_APP_DATABASE_URL,
+    databaseURL: "https://codelabz-58d31-default-rtdb.asia-southeast1.firebasedatabase.app",
```

Comment:
it's better to keep this take from env variables.

Reply:
By mistake, I committed it! Updated in the new commit